### PR TITLE
ref(ui): Simplify admin layout

### LINKED ISSUE #18866: Page contents below sidebar when switching from Admin
## Important Details

How are you running Sentry?

On-Premise w/ Docker, version 10.1.0.dev (09dc4ddf - latest on-premise)

## Description

When switching from Admin page (/manage/) to Projects or other pages, the page contents are partially obstructed by sidebar.
Screenshot of clean install: https://gyazo.com/081e59d0a994aff19b86c8688117ec9e

## Steps to Reproduce

1. Open Admin from submenu or navigate to /manage/
2. Click "Projects" on sidebar to switch to Projects page
3. Projects page contents are obstructed by sidebar

### What you expected to happen

Contents should not be obstructed.

### Possible Solution

There seems to be a missing class on <body> - "body-sidebar" - which adds padding to the body.

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Fixes GH-18866

### CODE REVIEW

File: src/sentry/static/sentry/app/views/admin/adminLayout.jsx [hunk 1]
```diff
@@ -1,67 +1,67 @@
 import DocumentTitle from 'react-document-title';
 import React from 'react';
+import styled from '@emotion/styled';
 
+import space from 'app/styles/space';
 import Hook from 'app/components/hook';
 import ListLink from 'app/components/links/listLink';
-import Sidebar from 'app/components/sidebar';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/admin/adminLayout.jsx:6]: Why get rid of sidebar?

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/admin/adminLayout.jsx:6]: It's rendered by the parent component. This actually did nothing but render twice. It being there was just wrong.

Comment (billyvg) [reply src/sentry/static/sentry/app/views/admin/adminLayout.jsx:6]: This is good info for the PR description

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/admin/adminLayout.jsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/admin/adminLayout.jsx:6]: yeah good point :)

**Comment by EvanPurkhiser** [comment]
CI was stalled so just merged another change that converts this to TS too 🤷

**Review by leedongwei — APPROVED** [review]

**Pull request merged**